pkg/vpcmodel/drawioOutput.go

```diff
@@ -177,10 +183,81 @@ func (d *DrawioOutputFormatter) createEdges() {
 	}
 }
 
+// explainableEndpoints() createExplanations() to be reimplement with real Shiri work
+func (d *DrawioOutputFormatter) createExplanations() []drawio.ExplanationEntry {
+	type expKey struct {
+		src, dst EndpointElem
+	}
+	explanations := map[expKey]string{}
+	allEndpoints := d.explainableEndpoints()
+	for _, src := range allEndpoints {
+		for _, dst := range allEndpoints {
+			explanations[expKey{src, dst}] = "No Connectivity from " + src.Name() + " to " + dst.Name()
+		}
+	}
+	for _, vpcConn := range d.conns {
+		for _, line := range vpcConn.GroupedLines {
+			srcs := []EndpointElem{line.src}
+			dsts := []EndpointElem{line.dst}
+			if srcList, ok := line.src.(*groupedEndpointsElems); ok {
+				srcs = *srcList
+			}
+			if dstList, ok := line.dst.(*groupedEndpointsElems); ok {
+				dsts = *dstList
+			}
+			for _, src := range srcs {
+				for _, dst := range dsts {
+					explanations[expKey{src, dst}] = line.String()
+				}
+			}
+		}
+	}
+	explanationsList := []drawio.ExplanationEntry{}
+	for k, e := range explanations {
+		explanationsList = append(explanationsList, drawio.ExplanationEntry{Src: d.gen.TreeNode(k.src), Dst: d.gen.TreeNode(k.dst), Text: e})
+	}
+	return explanationsList
+}
+
+func (d *DrawioOutputFormatter) explainableEndpoints() []EndpointElem {
+	subnetMode := d.uc == AllSubnets
+	allEndpoints := []EndpointElem{}
+	for _, vpcConfig1 := range d.cConfigs {
+		if !vpcConfig1.IsMultipleVPCsConfig {
+			if !subnetMode {
+				for _, n := range vpcConfig1.Nodes {
+					if !n.IsExternal() && d.showResource(n) {
+						allEndpoints = append(allEndpoints, n)
+					}
+				}
+			} else {
+				for _, s := range vpcConfig1.Subnets {
+					if !s.IsExternal() && d.showResource(s) {
```

Comment:
I don't believe a Subnet can be external

Reply:
fixed